installed-files-checker.py

```diff
@@ -0,0 +1,126 @@
+# AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO,
+# THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR
+# PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL APPLE INC. OR ITS CONTRIBUTORS
+# BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR
+# CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF
+# SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS
+# INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN
+# CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE)
+# ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF
+# THE POSSIBILITY OF SUCH DAMAGE.
+
+import sys
+from jinja2 import Environment, FileSystemLoader
+import argparse
+import os
+
+parser = argparse.ArgumentParser(description='Checker for Qtwebkit Binaries')
+parser.add_argument(
+    "--version", help=r"Version history of the form {major_version}.{minor_version}.{ver_patch}", required=True)
+parser.add_argument("--install_prefix", help="QtWebkit Install Prefix")
+parser.add_argument("--os", help="Operating system",
+                    required=True, choices=["linux", "macos", "windows"])
+parser.add_argument("--template", help='Relative path to template file',
+                    default="template/QtBinaryChecklist.txt")
+parser.add_argument("--release", help='Release build', action='store_true')
+parser.add_argument("--debug", help='Debug build', action='store_true')
+parser.add_argument("--qt_install_header", help='Qt headers install path')
+parser.add_argument("--qt_install_libs", help='Qt libraries install path')
+parser.add_argument("--qt_install_archdata", help='Qt archdata install path')
+parser.add_argument("--qt_install_libexecs", help='Qt libexecs install path')
+parser.add_argument("--force_debug_info",
+                    help='Enable debug symbols for release builds', action='store_true')
+parser.add_argument("--icu_version", help='ICU version')
+parser.add_argument(
+    "--toolchain", help='Toolchain used e.g. msvc, mingw for windows')
+
+args = parser.parse_args()
+
+if not args.release and not args.debug:
+    print("Please specify at least one build type!")
+    exit(1)
+
+template_abspath = os.path.abspath(args.template)
+template_folder = os.path.dirname(template_abspath)
+template_name = os.path.basename(template_abspath)
+
+file_loader = FileSystemLoader(template_folder)  # directory of template file
+env = Environment(loader=file_loader)
+
+template = env.get_template(template_name)  # load template file
+
+major, minor, patch = args.version.split('.')
+
+check_list = template.render(os=args.os,
+                             major=major, version=args.version, release=args.release, debug=args.debug,
+                             icu_version=args.icu_version, force_debug_info=args.force_debug_info, toolchain=args.toolchain).split('\n')
+
+
+def verify_linux(check_list):
```

Comment:
Seems like verify_linux should work for all platforms, if they don't customize --qt_install_headers etc. it will use defaults